src/include/session.h

```diff
@@ -197,8 +197,9 @@ struct __wt_session_impl {
 #define WT_SESSION_QUIET_CORRUPT_FILE 0x01000000u
 #define WT_SESSION_READ_WONT_NEED 0x02000000u
 #define WT_SESSION_RESOLVING_TXN 0x04000000u
-#define WT_SESSION_SCHEMA_TXN 0x08000000u
-#define WT_SESSION_SERVER_ASYNC 0x10000000u
+#define WT_SESSION_RTS 0x08000000u
```

Comment:
Is RTS rollback to stable? That's not obvious from the context here.

I feel like we need an abbreviation, but this doesn't seem like the right place to introduce one.

Reply:
Rename the macro to `WT_SESSION_ROLLBACK_TO_STABLE` for better understanding.